resources/init-postgres/create-db.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Can you please either bootstrap all data from init.sql or restore the init.sql script?